contrib/verify-commits/gpg.sh

```diff
@@ -9,7 +9,7 @@ REVSIG=false
 IFS='
 '
 if [ "$BITCOIN_VERIFY_COMMITS_ALLOW_SHA1" = 1 ]; then
-	GPG_RES="$(echo "$INPUT" | gpg --trust-model always "$@" 2>/dev/null)"
+	GPG_RES="$(echo "$INPUT" | "$GPG" --trust-model always "$@" 2>/dev/null)"
```

Comment:
Is this script ever called separately?
We should document in a comment that this script requires environment variable GPG to be set.
Also maybe it should fail if it is not.